src/Backend/EdgeTPU/EdgeTPUToolchain.ts

```diff
@@ -140,4 +143,30 @@ class EdgeTPUCompiler extends DebianCompiler {
   }
 }
 
-export { EdgeTPUDebianToolchain, EdgeTPUCompiler };
+class EdgeTPUToolchain implements Backend {
+  private readonly backendName: string;
+  private readonly toolchainCompiler: Compiler | undefined;
+
+  constructor() {
+    this.backendName = "EdgeTPU";
+    this.toolchainCompiler = undefined;
```

Comment:
I think this line should be changed to use EdgeTPUCompiler

```suggestion
    this.toolchainCompiler = new EdgeTPUCompiler();
```

Reply:
Thanks, I'll fix the code. 😄